packages/frontend/app/pods/components/pod/application/top-nav/template.hbs

```diff
@@ -1,3 +1,4 @@
+<head><title>emberclear {{this.unreadMessageCount}} {{this.currentUser.name}}</title></head>
```

Comment:
This looks like a good way to format the title, however:

The head tag may only live within the HTML tag's content. All app code lives in the body tag.

Since we can set the title in JavaScript via document.title, we should just do that. Don't need have this kind of thing in the template.

Just need a render modifier to call a function that set's the title